### ISSUE TITLE
Support tabbed content on print

### ISSUE THREAD

**Issue by timvink** [squidfunk/mkdocs-material#1974]
__I checked that...__

- [x] ... the documentation does not mention anything about my idea
- [x] ... to my best knowledge, my idea wouldn't break something for other users
- [x] ... there are no open or closed issues that are related to my idea

## Description

When printing, tabbed content does not display (either tab). Example (slate color theme):

![image](https://user-images.githubusercontent.com/5570380/95501747-11f00e00-09a9-11eb-9245-3966a8e4a2bb.png)

### Use Cases

When printing, content of all tabs should at least display. Ideally, the labels for each tab also move along with it. =

### Screenshots / Mockups

```css
@media print {
    .tabbed-content {
        display: block !important;
        padding-bottom: 1em;
    }
}
```

This provides a fix:

![image](https://user-images.githubusercontent.com/5570380/95502289-e7eb1b80-09a9-11eb-9ccc-fc8c6e61d6e2.png)

**Comment by squidfunk** [comment]
Nice solution, but we should also fix the tab order by resetting `order` on the content blocks 😄

**Comment by timvink** [comment]
> I'm not entirely sure whether this is the desired behavior for all use cases of tabbed content. Maybe sometimes it's desirable to only print the active tab.

Yeah fair point. But unless there is an option to configure this behaviour in `theme`, I would go for the most common use-case. I think displaying all content on print would make sense in most cases so that can be the default.

I also wonder if there are enough people with a use-case to hide tabbed content on print, to justify having an extra option in the theme to control this. Hard to assess 🤷‍♂️ 

I am convinced displaying tabbed content on print is a better default behaviour than hiding it.

**Comment by squidfunk** [comment]
Thanks for laying out your thoughts. I'm not really keen on providing and maintaining yet another option for such a "small" thing. Also, it seems kind of weird to provide an option only for printing.

> I am convinced displaying tabbed content on print is a better default behaviour than hiding it.

I hope you're right and share your opinion. It makes more sense to expand than to not do it. However, I'm sure there will be people complaining about the new behavior, but hey, can't save everyone. After all, this is an opinionated project, and I'm psyched the defaults seem to be sane, work so well and this project got so popular 😅

**Comment by squidfunk** [comment]
Fixed in c6a67ec0.

**Comment by squidfunk** [comment]
Released as part of 6.1.0.
